src/components/Error/Error.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { CSSProperties } from "react";
+import { Link, useHistory } from "react-router-dom";
+import Button from "components/common/Button";
+import { theme } from "styles/theme";
+import { ReactComponent as TaxiLogo } from "static/assets/TaxiLogo.svg";
+
+const Error = () => {
+  // const { error } = useParams<ErrorParams>();
+  const history = useHistory();
+
+  // FIXME Global State 에서 오류 가져오기
+  // const errorPrimaryState = error ? error : "404";
+  const stylePage: CSSProperties = {
+    display: "flex",
+    alignItems: "center",
+    justifyContent: "center",
+    flexDirection: "column",
+    height: "100%",
+  };
+  const styleLogo = { height: 54, marginBottom: 16 };
+  const styleTitle = { ...theme.font20, marginBottom: 12 };
+  const styleMessage = {
+    ...theme.font12,
+    color: theme.gray_text,
+    marginBottom: 24,
+  };
+
+  return (
+    <div style={stylePage}>
+      <TaxiLogo style={styleLogo} />
+      <div style={styleTitle}>예상치 못한 오류가 발생하였습니다</div>
+      {/* // FIXME 에러 메세지 만들기 */}
+      <div style={styleMessage}>아래 버튼을 클릭해주세요</div>
+      <div style={{ display: "flex", columnGap: "12px" }}>
+        <Button
+          type="white"
+          padding="13px 24px 14px"
+          radius={12}
+          font={theme.font14}
+          onClick={() => history.goBack()}
```

Comment:
인자가 없어 다음과 같이 사용할 수도 있을 것 같아요
```suggestion
          onClick={history.goBack}
```

Reply:
`Button` 컴포넌트에 `onClick`의 타입을 `() => void`로 정의해둬서 저렇게 했습니다. `history.goBack()`의 타입이 그냥 `void`이기도 하구요.

Reply:
아 제가 잘못 봤네요. 왜 그런지는 정확히 모르겠는데 괄호 없이 넣으면 타입이 맞네요!